dist_run.py

```diff
@@ -182,11 +284,12 @@ def main():
 
     # Distribute model on TP mesh
     model.distribute(tp_mesh)
-    logger.info(f"Model: {model}")
+    # logger.info(f"Model: {model}")
```

Comment:
temporary code change? (not intended to commit?)

Reply:
yes, was just adding to console noise during debugging...reverted.